packages/app/src/components/challenges/ImageUpload.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { ReactElement, useState } from 'react';
+import { Text, View, StyleSheet, Image, TouchableOpacity } from 'react-native';
+import ImagePicker from 'react-native-image-crop-picker';
+import { Button, colors } from '../UI';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+type Props = {};
```

Comment:
From now on, make sure to add an @TODO tag when there are things to come back and do